tsconfig.json

```diff
@@ -5,8 +5,19 @@
     "module": "commonjs",
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "allowJs": true,
-    "checkJs": true
+    "allowJs": true, // Allow JavaScript files to be compiled.
+    "checkJs": true, // Enable type checking on JavaScript files.
+    "noEmit": true, // Do not generate any files.
+    "strict": true, // Enable all strict type checking options.
+    "types": ["node"] // Include node type definitions.
```

Comment:
All `@types/*` pacakges are included in `types` by default, including `@types/node`. There's no need to manually add them.